projects/rero/ng-core/src/lib/record/search/aggregation/slider/slider.component.html

```diff
@@ -0,0 +1,28 @@
+<!--
+ RERO angular core
+ Copyright (C) 2020 RERO
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, version 3 of the License.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<mv-slider [(value)]="range" [min]="min" [max]="max" [range]="true" [step]="step"></mv-slider>
+<div class="flex-row d-flex justify-content-center mt-2">
+  <input type="number" [min]="min" [max]="max" class="form-control form-control-sm text-center" [(ngModel)]="range[0]" #ctrl="ngModel">
+  <span class="mx-3">-</span>
+  <input type="number" [min]="min" [max]="max" class="form-control form-control-sm text-center" [(ngModel)]="range[1]" #ctrl="ngModel">
+</div>
+<div class="text-center mt-2">
+  <small>
+    <a href class="mx-3" (click)="$event.preventDefault(); updateFilter()" translate>Show</a>
```

Comment:
why not put the `preventDefault` in the method?

Reply:
Because I think `updateFilter` method is not responsible for this behavior specific for templating. As this we can for example call the method from elsewhere in the component.

Reply:
You can also add `[routerLink]=""` instead of prevent default.